examples/data-sources/b1ddi_dns_forward_nsgs/data-source.tf

```diff
@@ -1,18 +1,25 @@
 terraform {
   required_providers {
     b1ddi = {
-      version = "0.1"
       source  = "infobloxopen/b1ddi"
     }
   }
 }
 
 # Get DNS Forward NSG with the specified name
-data "b1ddi_dns_forward_nsgs" "tf_example_forward_nsg" {
+data "b1ddi_dns_forward_nsgs" "tf_example_forward_nsgs" {
   filters = {
-    name = "tf_example_forward_nsg"
+    name = "tf_example_forward_nsgs"
```

Comment:
Same as above.

Reply:
Reverted it back to "tf_example_forward_nsg"